css/selectors/invalid.json

```diff
@@ -51,7 +51,8 @@
         },
         "form": {
           "__compat": {
-            "description": "Applies to <a href='https://developer.mozilla.org/docs/Web/HTML/Element/form'>&lt;form&gt;</a> elements",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/form",
```

Comment:
I think the link here might be confusing. Really, the feature relates to the selector, not the form element. On MDN, this link makes sense, but it might be confusing in other consumers' contexts

```suggestion
```